tool/site/client/src/components/common/prescriptionActions.tsx

```diff
@@ -1,70 +1,68 @@
 import {ActionLink} from "nhsuk-react-components"
-import * as React from "react"
-import {useContext} from "react"
+import React, {useContext} from "react"
 import {AppContext} from "../../index"
 
-interface PrescriptionActionsProps {
+export interface PrescriptionActionsProps extends Actions {
   prescriptionId: string
+}
+
+export interface Actions {
   cancel?: boolean
   release?: boolean
   verify?: boolean
   releaseReturn?: boolean
   dispense?: boolean
   withdraw?: boolean
   claim?: boolean
+  claimAmend?: boolean
   view?: boolean
 }
 
-const PrescriptionActions: React.FC<PrescriptionActionsProps> = ({
-  prescriptionId,
-  cancel,
-  release,
-  verify,
-  releaseReturn,
-  withdraw,
-  dispense,
-  claim,
-  view
-}) => {
+const PrescriptionActions: React.FC<PrescriptionActionsProps> = ({prescriptionId, ...actions}) => {
   const {baseUrl} = useContext(AppContext)
   return (
     <>
-      {cancel && (
+      {actions.cancel && (
```

Comment:
object destructuring ;) could pull them all from actions and then don't need all the `action.` here

Reply:
lol i changed from destructuring in the function definition because of so many lines